flow/layers/platform_view_layer.cc

```diff
@@ -16,6 +16,8 @@ void PlatformViewLayer::Preroll(PrerollContext* context,
                                     size_.height()));
 }
 
-void PlatformViewLayer::Paint(PaintContext& context) const {}
-
+void PlatformViewLayer::Paint(PaintContext& context) const {
+  EmbeddedViewParams params;
```

Comment:
How/when are these params populated? Right now, these are always empty.

Reply:
I left this for #6614